submissions/innazozulia/Hooli-style-popup/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
This PR contains work from the previous task with the popup. Please, make sure that you created this branch from the `master` branch (not from the popup task branch)